adapter/internal/adapter/adapter.go

```diff
@@ -373,13 +373,9 @@ func fetchChunkedAPIsOnStartUp(conf *config.Config, apiUUIDList []string, xdsOpt
 		} else {
 			adapter.GetAPIs(c, nil, envs, sync.RuntimeArtifactEndpoint, true, nil, queryParamMap)
 		}
-	} else {
-		if conf.ControlPlane.DynamicEnvironments.Enabled {
-			queryParamMap = common.PopulateQueryParamForDataPlane(queryParamMap)
-			adapter.GetAPIs(c, nil, nil, sync.RetrieveRuntimeArtifactEndpoint, true, apiUUIDList, queryParamMap)
-		} else {
-			adapter.GetAPIs(c, nil, envs, sync.APIArtifactEndpoint, true, apiUUIDList, queryParamMap)
-		}
+	} else if conf.ControlPlane.DynamicEnvironments.Enabled {
```

Comment:
We no longer need Dynamic Environments Enabled flag too.

Reply:
Config is used in multiple places. Since the issue -> https://github.com/wso2-enterprise/choreo/issues/31875
Is bit urgent shall we track removal of it with another task ?

<img width="922" alt="image" src="https://github.com/user-attachments/assets/9f0ac752-134c-4698-aef5-fc478e8026b2" />

</hr>

<img width="922" alt="image" src="https://github.com/user-attachments/assets/2d38bb50-46b5-4fcd-a416-c8bb15605387" />

Reply:
If we remove the code, wont the config get ignored or throw an error?

Reply:
You mean the cp-overlay config right ? 
I'll not throw an error. But we'll have to clean up the repos if we remove the config.

Reply:
> If we remove the code, wont the config get ignored or throw an error?

Ignored.
Since it is always TRUE (feature flag) we can think it's boolean true and remove unnecessary if blocks.